### ISSUE TITLE
Issue with WINGET

### ISSUE THREAD

**Issue by LucNanquette** [builtbybel/Winpilot#53]
Hi, your app provoked the following issue with `winget`. Running `winget upgrade -all` gave me this error
```
Failed when searching source: msstore
An unexpected error occurred while executing the command:
0x8a150044 : The rest source endpoint is not found.
```
After examining the logs, I found this:
```
[REPO] Finding available package from installed package using system reference search: Query:[none] Include:ProductCode='{1b9d7180-94ca-4b14-b20f-5adb5b87bd65}'[Exact] Include:UpgradeCode='{122135fb-9551-4b8f-af65-85eb2f192f9c}'[Exact] Include:NormalizedNameAndPublisher='bloatynosy'+'builtbybel'[Exact]
[REPO] Sending http GET request to: https://storeedgefd.dsx.mp.microsoft.com/v9.0/packageManifests/XPDCJG88N6FD16?Market=LU
[REPO] Response status: 404
[FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\Rest\Schema\HttpClientHelper.cpp(137)\WindowsPackageManager.dll!00007FFCE5A77EB3: (caller: 00007FFCE5A77818) Exception(1) tid(1ee0) 8A150044 
[FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\CompositeSource.cpp(993)\WindowsPackageManager.dll!00007FFCE5B70892: (caller: 00007FFCE5A4F883) LogHr(1) tid(1ee0) 8A150044     Msg:[D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\Rest\Schema\HttpClientHelper.cpp(137)\WindowsPackageManager.dll!00007FFCE5A77EB3: (caller: 00007FFCE5A77818) Exception(1) tid(1ee0) 8A150044 ] 

[REPO] Failed to search source for correlation: msstore
[REPO] Did not find Id [XPDCJG88N6FD16] in tracked source: msstore
```
Uninstalling your app solved the problem.
Hope this helps 😉

**Comment by Belim** [comment]
seems that the winget source is broken and msstore source is working 🤔

**Comment by Belim** [comment]
That has nothing to do with the app. What did you trigger with BloatyNosy earlier? Did you uninstall all the apps?

**Comment by LucNanquette** [comment]
Actually I did not use `BloatyNosy` yet. I installed it to have a look at what it offers. But I didn't "trigger" any of its functions yet.
I did some testing now. I reinstalled `BloatyNosy` latest version. Before the install no error with `winget upgrade`, but just after installing the error is back. I did not even run the app once. Same log as above. Uninstalling `BloatyNosy` and the error is gone.
So there is definitely a problem with `BloatyNosy` and the Microsoft Store.
